### PR TITLE
add option to allow encryption of root volume

### PR DESCRIPTION
**Pull request by mirkop-mattr**
Allow to configure the root volume to be encrypted at rest

### CODE REVIEW

**Comment by mirkop-mattr** [comment]
@perlboy FYI

File: modules/consul-cluster/variables.tf [hunk 1]
```diff
@@ -164,6 +164,11 @@ variable "root_volume_delete_on_termination" {
   default     = true
 }
 
+variable "root_volume_encrypted" {
+  description = "Encrypt the volume at rest"
```
Comment (brikis98) [inline modules/consul-cluster/variables.tf:168]:
```suggestion
  description = "Encrypt the root volume at rest"
```

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR! One tiny NIT and then I can kick off tests.

**Comment by mirkop-mattr** [comment]
> Thanks for the PR! One tiny NIT and then I can kick off tests.

@brikis98 I have made that change as suggested.

File: modules/consul-cluster/variables.tf [hunk 2]
```diff
@@ -164,6 +164,11 @@ variable "root_volume_delete_on_termination" {
   default     = true
 }
 
+variable "root_volume_encrypted" {
+  description = "Encrypt the root volume at rest"
+  default     = false
```
Comment (brikis98) [inline modules/consul-cluster/variables.tf:169]: Crap, sorry, one more thing I missed on the first go around:

```suggestion
  description = "Encrypt the root volume at rest"
  type        = bool
  default     = false
```

**Comment by mirkop-mattr** [comment]
@brikis98 PR updated. Apologies, my bad for not setting the type.

**Review by brikis98 — APPROVED** [review]
Thanks! I'll kick off tests now.

**Comment by brikis98** [comment]
The tests passed! Merging and releasing now.

**Comment by brikis98** [comment]
https://github.com/hashicorp/terraform-aws-consul/releases/tag/v0.8.1

**Pull request merged**